packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.Props.ts

```diff
@@ -45,7 +45,7 @@ export interface IComboBoxProps extends ISelectableDroppableTextProps<IComboBox>
    * Whether the ComboBox auto completes. As the user is inputing text, it will be suggested potential matches from the list of items. If
    * the combo box is expanded, this will also scroll to the suggested item, and give it a selected style. Defaults to false.
    */
-  autoComplete?: boolean;
+  autoComplete?: string;

```

Comment:
Why do we change to string? This is breaking change, can't we keep `boolean` and only map to `on` once we actually render?